src/graphics/GraphvizWiringDiagrams.jl

```diff
@@ -13,6 +13,7 @@ using StaticArrays
 import ...Theories: HomExpr
 using ...WiringDiagrams, ...WiringDiagrams.WiringDiagramSerialization
 using ...CategoricalAlgebra.CSets, ...Graphs, ..GraphvizGraphs
+import ...CategoricalAlgebra: migrate!
```

Comment:
Import is added but not used?

Reply:
Now it's used for data migration as mentioned above. I think I put this in originally because I had hopes that data migration would work for this, I'm glad it actually does!